### PR TITLE
Create Bot flow post and add images

### PR DESCRIPTION
**Pull request by psanchezelena**

### CODE REVIEW

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+Además, deberemos analizar en profundidad la identidad que podría adoptar.
+Una buena posibilidad es que dé respuestas y sugerencias en un tono divertido e ingenioso. El usuario obtendrá una experiencia positiva y la sensación de estar teniendo una conversación lo más fiel posible a la que tendría con un humano… divertido y afable 😊.
+En cualquier caso, esto queda a criterio de cada creador. 
+
+Por el momento, en este post nos centraremos en los **flujos conversacionales y cómo trasladarlos a LUIS**.
+
+## Muy brevemente, ¿Qué es LUIS?
+Aunque en el siguiente [enlace](https://docs.microsoft.com/es-es/azure/cognitive-services/LUIS/Home) podéis leeros toda la documentación, me gustaría explicar muy por encima cómo funciona LUIS.
+
+L.U.I.S **(Learn abour Language Understanding Intelligent Service)** es un servicio cognitivo creado por Microsoft con el fin de reconocer el lenguaje natural. Se trata de una herramienta que  analiza e interpreta las peticiónes de los usuarios. Sustrae ciertos datos e intenciones, que hemos configurado previamente, y nos devuelve una valoración (***score***), que analizaremos posteriormente.
```
Comment (franmolmedo) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: LUIS (Language Understanding Intelligent Service)

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
+![BotFlow](./images/FondoPostLUIS.png)
```
Comment (danimart1991) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: Remove "Learn Abour".

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+A continuación, vamos a analizar un diagrama diseñado a partir del flujo conversacional de este blog:
+
+![BotFlow](./images/FlowBotGeeksWPT-BR.jpg)
+
+
+
+
+El **flujo del diálogo inicial** sería el siguiente: 
+
+Primeramente, el bot debe comprobar si es la primera vez que van a interactuar, si es así, el bot manda un saludo de presentación. Si no lo es, se queda esperando a que el usuario solicite algo.
```
Comment (franmolmedo) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: Primero, o en primer lugar

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
+![BotFlow](./images/FondoPostLUIS.png)
+
+## Introducción
+
+Aunque ya hace como un año que Microsoft presentó **Bot Framework** en la **Build2016**, es en estos últimos meses cuando hemos percibido un notable crecimiento en el interés de diversos clientes en adquirir su propio bot-chat.
```
Comment (danimart1991) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: "Build2016" doesn't have genre. You could use "en su Microsoft Build2016"

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+A continuación, vamos a analizar un diagrama diseñado a partir del flujo conversacional de este blog:
+
+![BotFlow](./images/FlowBotGeeksWPT-BR.jpg)
+
+
+
+
+El **flujo del diálogo inicial** sería el siguiente: 
+
+En primer lugar, el bot debe comprobar si es la primera vez que van a interactuar, si es así, el bot manda un saludo de presentación. Si no lo es, se queda esperando a que el usuario solicite algo.
```
Comment (acnagrellos) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: SG: I usually don't check if it is the first time that bot talk. I prefer only put an intent as greeting

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+
+## Documentación
+
+Voy a dejar los modelos completos en nuestro [GitHub](https://github.com/WindowsPlatformTeam).
+
+[L.U.I.S (Language Understanding Intelligent Service)](https://docs.microsoft.com/es-es/azure/cognitive-services/LUIS/Home)
+
+[Prebuilt entities disponibles por idiomas](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/pre-builtentities)
+
+
```
Comment (acnagrellos) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: SG: You can add here Postman link and BotFramework documentation link.

File: elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+[Documentación L.U.I.S (Language Understanding Intelligent Service)](https://docs.microsoft.com/es-es/azure/cognitive-services/LUIS/Home)
+
+[Crear una cuenta luis.ai](luis.ai)
+
+[Prebuilt entities disponibles por idiomas](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/pre-builtentities)
+
+[Documentación BotFramework](https://dev.botframework.com/)
+
+[Postman](https://www.getpostman.com/)
+
```
Comment (maktub82) [inline elenaps/Code Flow & L.U.I.S/BotFlow&LUIS.md]: Añade una conclusión/resumen de lo que hemos hecho. Habla de los siguientes pasos y despídete.

**Pull request merged**